draft-vasters-json-structure-core.md

```diff
@@ -1051,6 +1051,36 @@ schema definition, including in type unions.
 `$ref` is NOT permitted in other attributes and MUST NOT be used inside the
 `type` of the root object.
 
+### `id` Keyword {#object-id-keyword}
+
+The `id` keyword is only applicable on objects and states the properties that
+make up the primary ID(s) of the object. Providing more than one ID indicates
```

Comment:
lists the properties, in the role of identifiers,  that establish an unambiguous identity of an instance. Providing more than one property reference indicates a composite identity, not a list of alternate identifiers.